### DISCUSSION TITLE
🚀 Feature request: Allow Toolbar selection when setting Wygwam field as a Pro Variable

### DISCUSSION THREAD

**Discussion by pyrobob83** [ExpressionEngine/ExpressionEngine#3291] [Feature Requests]
### Suggest an idea for ExpressionEngine

Right now, if you create a Pro Variable with a Wygwam field, you can't specify which toolbar it should use.  The dirty workaround is to set the primary variable type as a Grid and just have a single column with type Wygwam - that method lets you choose the toolbar, but I imagine I won't get any rebuttal for suggesting it should be available to a standalone Wygwam variable.

**Comment by intoeetive** [comment]
Since this is is related to Wygwam and not ExpressionEngine, I'll pass it on to Wygwam team but will close the discussion. Thank you!
